### ISSUE TITLE
How can I know that the accuracy is good or not using RMSE for regression tasks?

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#34123]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Windows 10 x64
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):binary
- TensorFlow version (use command below):1.13.1
- Python version:3.4.6
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:9.0/7.4
- GPU model and memory:Nvidia Geforce 840m

**Describe the current behavior**
Hello, 
My question is very simple, I have trained a model for regression task ( detect eye region with landmarks ) using training dataset of 160000 images and evaluate it after 50000 steps using eval_mse that based on `tf.metrics.root_mean_squared_error`.
I would like to know how can I know that the accuracy is good by viewing the value of the RMSE using the `Eval_mode` and `Tensorboard`

That was some screenshots:
![losse](https://user-images.githubusercontent.com/19480228/68531460-91e1a400-0312-11ea-8352-380751600678.PNG)

![graphhh](https://user-images.githubusercontent.com/19480228/68531464-9908b200-0312-11ea-822c-987a88f1528b.PNG)

How can I understand the value of the `eval_mse` and the scalar of the RMSE?

**Comment by gowthamkpr** [comment]
@abdou31 Please refer to the following [link](https://stats.stackexchange.com/questions/56302/what-are-good-rmse-values) to understand what RMSE values are good for your problem.

Also please post questions like these in stack overflow as github is only meant for bug/performance, build/install and docs related issues. Thanks!
